management-api-agent-3.x/src/test/java/com/datastax/mgmtapi/rpc/ObjectSerializer3xTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+package com.datastax.mgmtapi.rpc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class ObjectSerializer3xTest {
```

Comment:
Issue:
We should probably duplicate this in all the agent projects as well. There might be a way to only write a single test that is run for each version, but if it's faster, just duplicate the test in each project and maybe write a refactor ticket to consolidate in the future.